dist/tools/codespell/ignored_words.txt

```diff
@@ -125,3 +125,15 @@ sur
 
 # unsecure (unlocked, protection disabled) => insecure
 unsecure
+
+# nce (neighbor cache entry) => once, nice
+nce
+
+# warmup => warm up, warm-up
+warmup
+
+# clen => clean, clan
+clen
```

Comment:
Do you have an idea why this was the only variable name that `codespell` is allergic to? I am very sure there are more variable names that are not an english word.

Reply:
I have no idea - I'm also wondering why those start popping up all of the sudden.